app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] **Optional:** All methods to update a counter in the different models could be set to private and used with `after_save` active callback.